client/index.html

```diff
@@ -214,12 +214,26 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Visual Aids</h2>
 							</div>
-							<div class="col-sm-12">
+							<div class="col-sm-6">
 								<label class="opt">
 									<input type="checkbox" name="colors">
 									Enable colored nicknames
 								</label>
 							</div>
+							<div class="col-sm-6">
+								<label class="opt">
+									<select id="themeselect" name="themeselect">
+									<% themes.forEach(function(theme) { %>
+										<option value="<%= theme.path %>" 
+										<% if (theme.default) { %>
+										selected
```

Comment:
What about `default` not being a property on the theme (but maybe a different one outside of `themes`, like `defaultTheme`), so that we avoid situations with multiple defaults?